### PR TITLE
Vectorized plots accept lists of col//labels

### PR DESCRIPTION
**Pull request by kylebaron**
Using `[[` to subset should work for both list and vector. 

The original idea was to implement for displays (e.g., `npde_covariate()`) but expanded to accept lists for all vectorized plots. 

# Example

``` r
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union
library(pmplots)
#> Loading required package: ggplot2
library(purrr)
library(patchwork)
library(yspec)

spec <- ys_help$spec()
co <- spec %>% ys_select(WT, AGE) %>% axis_col_labs() %>% as.list()

data <- ys_help$data() %>% mutate(NPDE = rnorm(n()), CWRES = NPDE)

npde_covariate(data, co)
#> `geom_smooth()` using formula = 'y ~ x'
#> `geom_smooth()` using formula = 'y ~ x'
```

``` r

npde_cont(data, co)
#> [[1]]
#> `geom_smooth()` using formula = 'y ~ x'
```

    #> 
    #> [[2]]
    #> `geom_smooth()` using formula = 'y ~ x'

<sup>Created on 2024-07-10 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

### CODE REVIEW

File: tests/testthat/test-col_label.R [hunk 1]
```diff
@@ -48,3 +48,17 @@ test_that("get col part from a col-label vector", {
...
+  b <- col_label(as.list(x))
+  expect_identical(a,b)
+})
+
+test_that("col_labels accepts list or vector", {
+  x <- c("a//b", "c@@d")
+  a <- pmplots:::col_labels(x)
+  b <- pmplots:::col_labels(as.list(x))
+  expect_identical(a,b)
+})
```
Comment (kyleam) [inline tests/testthat/test-col_label.R:64]: Looks like duplicate of test on lines 52-57 (aside from `pmplots:::` qualification, which I don't think is relevant).

**Review by kyleam — APPROVED** [review]
LGTM.  Left comment about repeated test.

Failing CI is due to the R 3.6 job and is resolved in `main` (with merge of gh-97).

File: tests/testthat/test-col_label.R (hunk 1 above)
Comment (kylebaron) [reply tests/testthat/test-col_label.R:64]: Thanks, @kyleam.   

The previous test (on 52-57) was for `col_label()` (exported) and this one is for a vectorized version of that (`col_labels()`) which isn't exported.  I could probably change `col_label()` to take input with length > 1, but it is all over the code and I didn't want to risk something. 

Ok to go ahead with tests as is?

Comment (kyleam) [reply tests/testthat/test-col_label.R:64]: Bah, sorry for overlooking that.  Thanks for the correction.

> Ok to go ahead with tests as is?

Yes 🤦

Comment (kylebaron) [reply tests/testthat/test-col_label.R:62]: Took me several minutes to figure that out too. :(

**Pull request merged**